test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -12,6 +12,8 @@ Could not find tests for function sys.babelfish_try_cast_floor_bigint
 Could not find tests for function sys.babelfish_try_cast_floor_int
 Could not find tests for function sys.babelfish_try_cast_floor_smallint
 Could not find tests for function sys.babelfish_try_cast_to_any
+Could not find tests for function sys.babelfish_try_cast_to_varbinary
+Could not find tests for function sys.babelfish_try_cast_to_varchar
```

Comment:
Do we have any dependent tests added for TRY_CAST when casting to `varbinary` and `varchar` ? if yes kindly point me to those tests.